compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -1092,15 +1081,6 @@ inline section_iterator *unwrap(LLVMSectionIteratorRef SI) {
   return reinterpret_cast<section_iterator *>(SI);
```

Comment:
I just noticed that the comment above this function was outdated a bit even before this PR, but will become plain confusing after the removal of `LLVMRustGetSectionName` in this PR.